app/views/uploads/_population_criteria.html.erb

```diff
@@ -0,0 +1,40 @@
+<div class="panel panel-default <%=population[:type]%>_children">
+  <div class="panel-heading">
+    <h4 class="panel-title panel-population">
+      <a data-toggle="collapse" href="#collapse_<%=measure[:cms_id]%>_<%=measure[:sub_id]%>_<%=population[:hqmf_id]%>">
+        <div>
+          <% title = population[:title] %>
+          <% if (population[:type] == 'OBSERV') then title = 'Observations' end %>
+          <span class="population-label <%=title%> rationale eval-<%=@rationale[population[:type]]%>"><b><%=title%></b><span class="sr-only sr-highlight-status"></span></span>
+          <div class="pull-right">
+            <i class="fa fa-lg fa-angle-down toggle-icon" aria-hidden="true"></i>
+            <span class="sr-only">toggle <%=title%></span>
+          </div>
+        </div>
+      </a>
+    </h4>
+  </div>
+
+  <div id="collapse_<%=measure[:cms_id]%>_<%=measure[:sub_id]%>_<%=population[:hqmf_id]%>" class="panel-collapse collapse in">
+    <div class="panel-body">
+      <% if population[:preconditions] %>
```

Comment:
Do you need to check if `population[:preconditions]` is empty or will there always be something there?